### DISCUSSION TITLE
orders.getOrders sorting?

### DISCUSSION THREAD

**Discussion by phstc** [amzn/selling-partner-api-models#3393] [General]
I'm polling `Shipped` orders through [`orders.getOrders`](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#getorders) and I'm querying using `LastUpdatedAfter` to poll forward, for example, after polling I get the latest `order.LastUpdateDate` save, then next time, I query query for `LastUpdatedAfter: lastPolledLastUpdateDate`.

I noticed that `LastUpdatedAfter` seems to be inclusive. It also includes the last order I polled with the `LastUpdateDate` matching the `lastPolledLastUpdateDate`.

My questions:

* I'm not paginating returned orders. I'm polling ten at a time. Can I rely that `getOrders` will be sorted by `LastUpdateDate`, or do I also need to save `NextToken` for my next poll? 
* Any other gotcha with this mechanism? 

I'm polling orders instead of using the Notifications API because of some infrastructure limitations.

**Comment by GJ-JG** [comment]
Hi @phstc , we released a youtube video maybe similar to this topic: https://www.youtube.com/watch?v=aeSAo5a6ClQ&t=24s.
In your case, currently in our documentation, no specifications can be found that the orders fetched by getOrders are sorted by ```lastUpdatedDate```, so we recommend using the ```nextToken``` for your next poll just in case of duplicated orders.

**Comment by phstc** [comment]
Hi @GJ-JG 

I ended up changing the poller to set the last update with the value of the last updated order returned. 

But the way described in the video seems more effective in avoiding redundant requests. 

Thanks for sharing.
